_sass/elements/_buttons.scss

```diff
@@ -150,6 +150,10 @@ button,
     font-size: 1.9rem;
     padding: 1.5rem 3rem;
   }
+
+  &.usa-text-capitalize {
+    text-transform: capitalize;
```

Comment:
I noticed this is capitalizing words we probably don't want to capitalize (e.g., `to`). I think we may need to use js to capitalize the language in the buttons instead of CSS. Can you add a TODO here to refactor that later?